src/components/DpFlyout/DpFlyout.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+
+import DpFlyout from './DpFlyout.vue'
+
+const meta: Meta<typeof DpFlyout> = {
+    component: DpFlyout,
+    parameters: {
+        docs: {
+            description: {
+                component: 'The Flyout component renders a flyout that is intended to show further content.\n' +
+                    '## Roadmap.\n' +
+                    '- use https://popper.js.org/docs/v2/ for positioning instead of basic css - or find another way to make left/right offset dynamic.'
```

Comment:
Does this also work?

```suggestion
                component: `
                    The Flyout component renders a flyout that is intended to show further content.
                    ## Roadmap
                    - use https://popper.js.org/docs/v2/ for positioning instead of basic CSS, or find another way to make left/right offset dynamic.`
```

Reply:
Looks like code, I wouldn't do that! What is your opinion?

![Screenshot 2023-07-25 at 12-14-16 DpFlyout - Docs ⋅ Storybook](https://github.com/demos-europe/demosplan-ui/assets/59571234/608b9202-b166-4b91-9fb2-5b8ed5caec11)

Reply:
Well, that does not look right.

However it might be cumbersome to drop each single line of our existing markdown docs into Meta.component like that.

As i see now, [Storybook 7 uses mdx2](https://storybook.js.org/docs/vue/migration-guide#upgrade-mdx1-to-mdx2) as default. Maybe we should try to get this running instead of the tsx implementation?

Reply:
with this fix can see markdown docs https://github.com/demos-europe/demosplan-ui/pull/395/commits/cf21974abaa091be4df3333e91f8380fadcc365e

@salisdemos @spiess-demos 

![Screenshot 2023-07-25 at 15-35-12 Components _ Flyout - Docs ⋅ Storybook](https://github.com/demos-europe/demosplan-ui/assets/59571234/a45692bb-c20f-4859-9a69-15c7fccd4aee)